client/state/action-types.js

```diff
@@ -194,7 +194,12 @@ export const KEYRING_SERVICES_REQUEST_SUCCESS = 'KEYRING_SERVICES_REQUEST_SUCCES
 export const LAYOUT_FOCUS_SET = 'LAYOUT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_ACTIVATE = 'LAYOUT_NEXT_FOCUS_ACTIVATE';
+export const MEDIA_ITEMS_RECEIVE = 'MEDIA_ITEMS_RECEIVE';
```

Comment:
My point was that by dispatching a separate single item action we could avoid boilerplate code like this: https://github.com/Automattic/wp-calypso/pull/8857/files#diff-ac2308bd4d1d8c2075ad658ea3b46b5bR60.

But I think your point makes sense, more actions could overcomplex the implementations in some cases. Feel free to ignore my comment.